app/services/analytics_events.rb

```diff
@@ -3415,6 +3415,10 @@ def piv_cac_login(success:, errors:, **extra)
     )
   end
 
+  def piv_cac_login_visited
+    track_event('PIV/CAC Login visited')
```

Comment:
I'm conflicted on the casing here, but we've got so much inconsistency already with these PIV-related events 🙈 

At the very least, I'd think we'd want to be consistent on casing between "login" and "visited"

Either:

```suggestion
    track_event('PIV/CAC Login Visited')
```
Or:

```suggestion
    track_event('PIV/CAC login visited')
```

Reply:
to make things more confusing, the event below (and the one two above) has "PIV CAC" no slash...

```suggestion
    track_event('PIV CAC Login visited')
```

Reply:
Another option, would be to rename them all to have consistent prefixes and leave some `@identity.idp.previous_event_name` tags to be kind to future log spelunkers?

Reply:
I think it might be good to have them to be consistent, I think it may break some current Piv/Cac logging.